monitor/linuxmonitor/processor.go

```diff
@@ -97,52 +88,14 @@ func (s *LinuxProcessor) Start(eventInfo *rpcmonitor.EventInfo) error {
 		return perr
 	}
 
-	// if eventInfo.PUID == "1" {
-	// 	markval, ok := runtimeInfo.Options().Get(cgnetcls.CgroupMarkTag)
-	// 	if !ok {
-	// 		return errors.New("Mark value not found")
-	// 	}
-	// 	mark, _ := strconv.ParseUint(markval, 10, 32)
-	// 	hexmark := "0x" + (strconv.FormatUint(mark, 16))
-	// 	if err := ioutil.WriteFile("/sys/fs/cgroup/net_cls,net_prio/net_cls.classid", []byte(hexmark), 0644); err != nil {
-	// 		return errors.New("Failed to  write to net_cls.classid file for new cgroup")
-	// 	}
-	// 	return nil
-	// }
```

Comment:
Can you tell why is it commented, not removed?

Reply:
Its removed .. you are looking at the removed parts.

Reply:
I was tired :)